src/Simplify.elm

```diff
@@ -4132,32 +4132,37 @@ resultMapErrorChecks checkInfo =
             else
                 []
         , \() ->
-            case Maybe.andThen (AstHelpers.getSpecificFunctionCall ( [ "Result" ], "Err" ) checkInfo.lookupTable) maybeResultArg of
```

Comment:
Personal opinion: Having less indentation and less case branches by using `Maybe.andThen` is actually better (as long as we don't need the intermediate value).

Reply:
How about a rule like "if it fits on one line it's fine to keep the `case andThen of`?"